src/detect-pangram.js

```diff
@@ -15,7 +15,16 @@
  * @return {boolean}
  */
 function detectPangram(phrase) {
-  // write code here
+  const templatePanagram = 'abcdifghijklmnopqrstuvwxyz';
```

Comment:
that`s the alphabet, yes?
```suggestion
  const alphabet = 'abcdifghijklmnopqrstuvwxyz';
```

Reply:
fixed